### ISSUE TITLE
Latest 15 Papers - April 30, 2025

### ISSUE THREAD

**Issue by KMnO4-zx** [KMnO4-zx/DailyArXiv#12]
**Please check the [Github](https://github.com/KMnO4-zx/DailyArXiv) page for a better reading experience and more papers.**

## Geoscience
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[GeoRSMLLM: A Multimodal Large Language Model for Vision-Language Tasks in Geoscience and Remote Sensing](http://arxiv.org/abs/2503.12490v1)** | 2025-03-16 | [papers-cool](https://papers.cool/arxiv/2503.12490v1) |  |
| **[Towards Foundation Models: Evaluation of Geoscience Artificial Intelligence with Uncertainty](http://arxiv.org/abs/2501.14809v1)** | 2025-01-15 | [papers-cool](https://papers.cool/arxiv/2501.14809v1) |  |
| **[When Geoscience Meets Foundation Models: Towards General Geoscience Artificial Intelligence System](http://arxiv.org/abs/2309.06799v5)** | 2024-11-12 | [papers-cool](https://papers.cool/arxiv/2309.06799v5) | <details><summary>accpe...</summary><p>accpeted by IEEE Geoscience and Remote Sensing Magazine</p></details> |
| **[Prototype-Based Methods in Explainable AI and Emerging Opportunities in the Geosciences](http://arxiv.org/abs/2410.19856v1)** | 2024-10-22 | [papers-cool](https://papers.cool/arxiv/2410.19856v1) | <details><summary>Accep...</summary><p>Accepted at AI for Science Workshop-Oral (Attention Track), Proceedings of 41st International Conference on Machine Learning (ICML) 2024</p></details> |
| **[Interpretable Geoscience Artificial Intelligence (XGeoS-AI): Application to Demystify Image Recognition](http://arxiv.org/abs/2311.04940v2)** | 2024-05-07 | [papers-cool](https://papers.cool/arxiv/2311.04940v2) | <details><summary>there...</summary><p>there are some erros in the results, and a newer revision is still preparing</p></details> |
| **[GeoGalactica: A Scientific Large Language Model in Geoscience](http://arxiv.org/abs/2401.00434v2)** | 2024-04-13 | [papers-cool](https://papers.cool/arxiv/2401.00434v2) |  |
| **[A novel approach for estimating functions in the multivariate setting based on an adaptive knot selection for B-splines with an application to a chemical system used in geoscience](http://arxiv.org/abs/2306.00686v3)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2306.00686v3) | 29 pages, 27 figures |
| **[When Geoscience Meets Generative AI and Large Language Models: Foundations, Trends, and Future Challenges](http://arxiv.org/abs/2402.03349v1)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2402.03349v1) |  |
| **[Differentiable modeling to unify machine learning and physical models and advance Geosciences](http://arxiv.org/abs/2301.04027v2)** | 2023-12-27 | [papers-cool](https://papers.cool/arxiv/2301.04027v2) |  |
| **[K2: A Foundation Language Model for Geoscience Knowledge Understanding and Utilization](http://arxiv.org/abs/2306.05064v2)** | 2023-09-13 | [papers-cool](https://papers.cool/arxiv/2306.05064v2) |  |
| **[AI Security for Geoscience and Remote Sensing: Challenges and Future Trends](http://arxiv.org/abs/2212.09360v2)** | 2023-06-22 | [papers-cool](https://papers.cool/arxiv/2212.09360v2) |  |
| **[Coarse-to-fine Task-driven Inpainting for Geoscience Images](http://arxiv.org/abs/2211.11059v3)** | 2022-12-06 | [papers-cool](https://papers.cool/arxiv/2211.11059v3) |  |
| **[Investigating the fidelity of explainable artificial intelligence methods for applications of convolutional neural networks in geoscience](http://arxiv.org/abs/2202.03407v2)** | 2022-09-05 | [papers-cool](https://papers.cool/arxiv/2202.03407v2) |  |
| **[Data Centred Intelligent Geosciences: Research Agenda and Opportunities, Position Paper](http://arxiv.org/abs/2209.02384v1)** | 2022-08-20 | [papers-cool](https://papers.cool/arxiv/2209.02384v1) |  |
| **[Beyond Visuals : Examining the Experiences of Geoscience Professionals With Vision Disabilities in Accessing Data Visualizations](http://arxiv.org/abs/2207.13220v1)** | 2022-07-27 | [papers-cool](https://papers.cool/arxiv/2207.13220v1) |  |

## Spatial
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[SpatialReasoner: Towards Explicit and Generalizable 3D Spatial Reasoning](http://arxiv.org/abs/2504.20024v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.20024v1) | <details><summary>Proje...</summary><p>Project page: https://spatial-reasoner.github.io</p></details> |
| **[STCOcc: Sparse Spatial-Temporal Cascade Renovation for 3D Occupancy and Scene Flow Prediction](http://arxiv.org/abs/2504.19749v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19749v1) |  |
| **[GPA-RAM: Grasp-Pretraining Augmented Robotic Attention Mamba for Spatial Task Learning](http://arxiv.org/abs/2504.19683v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19683v1) |  |
| **[Generalizing Egocentric Temporal Neighborhoods to probe for spatial correlations in temporal networks and infer their topology](http://arxiv.org/abs/2501.16070v2)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2501.16070v2) |  |
| **[Bullet: Boosting GPU Utilization for LLM Serving via Dynamic Spatial-Temporal Orchestration](http://arxiv.org/abs/2504.19516v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19516v1) |  |
| **[Frequency Domain Resampling for Gridded Spatial Data](http://arxiv.org/abs/2504.19337v1)** | 2025-04-27 | [papers-cool](https://papers.cool/arxiv/2504.19337v1) |  |
| **[Spatial-Sign based High dimensional Change Point Inference](http://arxiv.org/abs/2504.19306v1)** | 2025-04-27 | [papers-cool](https://papers.cool/arxiv/2504.19306v1) |  |
| **[RadioFormer: A Multiple-Granularity Radio Map Estimation Transformer with 1\textpertenthousand Spatial Sampling](http://arxiv.org/abs/2504.19161v1)** | 2025-04-27 | [papers-cool](https://papers.cool/arxiv/2504.19161v1) |  |
| **[Sparse VideoGen: Accelerating Video Diffusion Transformers with Spatial-Temporal Sparsity](http://arxiv.org/abs/2502.01776v2)** | 2025-04-27 | [papers-cool](https://papers.cool/arxiv/2502.01776v2) | <details><summary>17 pa...</summary><p>17 pages, 11 figures, 3 tables</p></details> |
| **[Adaptation using spatially distributed Gaussian Processes](http://arxiv.org/abs/2312.14130v2)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2312.14130v2) | 90 pages, 6 figures |
| **[A Unified Approach for Multi-Granularity Search over Spatial Datasets](http://arxiv.org/abs/2412.04805v3)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2412.04805v3) |  |
| **[Real-time High-fidelity Gaussian Human Avatars with Position-based Interpolation of Spatially Distributed MLPs](http://arxiv.org/abs/2504.12909v2)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2504.12909v2) | <details><summary>CVPR ...</summary><p>CVPR 2025. Project page https://gapszju.github.io/mmlphuman/ . Code https://github.com/1231234zhan/mmlphuman</p></details> |
| **[LiLIS: Enhancing Big Spatial Data Processing with Lightweight Distributed Learned Index](http://arxiv.org/abs/2504.18883v1)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2504.18883v1) |  |
| **[GeoConformal prediction: a model-agnostic framework of measuring the uncertainty of spatial prediction](http://arxiv.org/abs/2412.08661v3)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2412.08661v3) |  |
| **[Spatial Speech Translation: Translating Across Space With Binaural Hearables](http://arxiv.org/abs/2504.18715v1)** | 2025-04-25 | [papers-cool](https://papers.cool/arxiv/2504.18715v1) | Accepted by CHI2025 |

## Time
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Real-Time Imitation of Human Head Motions, Blinks and Emotions by Nao Robot: A Closed-Loop Approach](http://arxiv.org/abs/2504.19985v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19985v1) |  |
| **[Feelbert: A Feedback Linearization-based Embedded Real-Time Quadrupedal Locomotion Framework](http://arxiv.org/abs/2504.19965v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19965v1) | <details><summary>Submi...</summary><p>Submitted to IEEE Transactions on Robotics</p></details> |
| **[On Stopping Times of Power-one Sequential Tests: Tight Lower and Upper Bounds](http://arxiv.org/abs/2504.19952v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19952v1) | 36 pages |
| **[Interpretable additive model for analyzing high-dimensional functional time series](http://arxiv.org/abs/2504.19904v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19904v1) |  |
| **[Promoting Real-Time Reflection in Synchronous Communication with Generative AI](http://arxiv.org/abs/2504.15647v2)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.15647v2) | <details><summary>Prese...</summary><p>Presented at the 2025 ACM Workshop on Human-AI Interaction for Augmented Reasoning, Report Number: CHI25-WS-AUGMENTED-REASONING</p></details> |
| **[Optimal real-time dynamic treatment regimes with application to oxytocin use in preventing postpartum hemorrhage](http://arxiv.org/abs/2504.19831v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19831v1) |  |
| **[SILENT: A New Lens on Statistics in Software Timing Side Channels](http://arxiv.org/abs/2504.19821v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19821v1) |  |
| **[Generative AI Act II: Test Time Scaling Drives Cognition Engineering](http://arxiv.org/abs/2504.13828v3)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.13828v3) | <details><summary>v3: a...</summary><p>v3: add the comparison to existing work part; fix some errors</p></details> |
| **[AIJIM: A Scalable Model for Real-Time AI in Environmental Journalism](http://arxiv.org/abs/2503.17401v5)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2503.17401v5) | <details><summary>22 pa...</summary><p>22 pages, 10 figures, 5 tables. Keywords: Artificial Intelligence, Environmental Journalism, Real-Time Reporting, Vision Transformers, Image Recognition, Crowdsourced Validation, GPT-4, Automated News Generation, GIS Integration, Data Privacy Compliance, Explainable AI (XAI), AI Ethics, Sustainable Development</p></details> |
| **[Multimodal Conditioned Diffusive Time Series Forecasting](http://arxiv.org/abs/2504.19669v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19669v1) |  |
| **[GAN-SLAM: Real-Time GAN Aided Floor Plan Creation Through SLAM](http://arxiv.org/abs/2504.19653v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19653v1) | <details><summary>10 pa...</summary><p>10 pages, preprint conference submission</p></details> |
| **[ARMOR: Adaptive Meshing with Reinforcement Optimization for Real-time 3D Monitoring in Unexposed Scenes](http://arxiv.org/abs/2504.19624v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19624v1) |  |
| **[A Comparative Study on Positional Encoding for Time-frequency Domain Dual-path Transformer-based Source Separation Models](http://arxiv.org/abs/2504.19605v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19605v1) | <details><summary>5 pag...</summary><p>5 pages, 3 tables, 2 figures</p></details> |
| **[Near-real-time flood inundation monitoring by Bayesian analysis for change point problems for Sentinel-1 time series](http://arxiv.org/abs/2504.19526v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19526v1) |  |
| **[Discrete-time Two-Layered Forgetting RLS Identification under Finite Excitation](http://arxiv.org/abs/2504.19518v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19518v1) | 6 pages, 6 figures |

## Trajectory
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Kinodynamic Trajectory Following with STELA: Simultaneous Trajectory Estimation & Local Adaptation](http://arxiv.org/abs/2504.20009v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.20009v1) | [Accepted] RSS 2025 |
| **[Towards Ball Spin and Trajectory Analysis in Table Tennis Broadcast Videos via Physically Grounded Synthetic-to-Real Transfer](http://arxiv.org/abs/2504.19863v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19863v1) | <details><summary>To be...</summary><p>To be published in 2025 IEEE/CVF International Conference on Computer Vision and Pattern Recognition Workshops (CVPRW)</p></details> |
| **[UTTG_ A Universal Teleoperation Approach via Online Trajectory Generation](http://arxiv.org/abs/2504.19736v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19736v1) |  |
| **[Geo-Llama: Leveraging LLMs for Human Mobility Trajectory Generation with Spatiotemporal Constraints](http://arxiv.org/abs/2408.13918v4)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2408.13918v4) |  |
| **[ARTEMIS: Autoregressive End-to-End Trajectory Planning with Mixture of Experts for Autonomous Driving](http://arxiv.org/abs/2504.19580v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19580v1) |  |
| **[An End-to-End Framework for Optimizing Foot Trajectory and Force in Dry Adhesion Legged Wall-Climbing Robots](http://arxiv.org/abs/2504.19448v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19448v1) |  |
| **[Trajectory Planning with Model Predictive Control for Obstacle Avoidance Considering Prediction Uncertainty](http://arxiv.org/abs/2504.19193v1)** | 2025-04-27 | [papers-cool](https://papers.cool/arxiv/2504.19193v1) | <details><summary>This ...</summary><p>This work has been accepted to IFAC for publication under a Creative Commons Licence CC-BY-NC-ND</p></details> |
| **[Efficient Control Allocation and 3D Trajectory Tracking of a Highly Manoeuvrable Under-actuated Bio-inspired AUV](http://arxiv.org/abs/2504.19049v1)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2504.19049v1) |  |
| **[Learning Stochastic Thermodynamics Directly from Correlation and Trajectory-Fluctuation Currents](http://arxiv.org/abs/2504.19007v1)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2504.19007v1) | <details><summary>11 pa...</summary><p>11 pages, 6 appendices (10 pages), 4 figures; https://csc.ucdavis.edu/~cmg/compmech/pubs/currents.htm</p></details> |
| **[DDAT: Diffusion Policies Enforcing Dynamically Admissible Robot Trajectories](http://arxiv.org/abs/2502.15043v2)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2502.15043v2) |  |
| **[A Unified Longitudinal Trajectory Dataset for Automated Vehicle](http://arxiv.org/abs/2406.00009v2)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2406.00009v2) | NA |
| **[Enhancing System Self-Awareness and Trust of AI: A Case Study in Trajectory Prediction and Planning](http://arxiv.org/abs/2504.18421v1)** | 2025-04-25 | [papers-cool](https://papers.cool/arxiv/2504.18421v1) | <details><summary>Accep...</summary><p>Accepted to be published as part of the 2025 IEEE Intelligent Vehicles Symposium (IV), JCluj-Napoca, Romania, June 22-25, 2025</p></details> |
| **[Equi-Euler GraphNet: An Equivariant, Temporal-Dynamics Informed Graph Neural Network for Dual Force and Trajectory Prediction in Multi-Body Systems](http://arxiv.org/abs/2504.13768v3)** | 2025-04-25 | [papers-cool](https://papers.cool/arxiv/2504.13768v3) | <details><summary>Reupl...</summary><p>Reuploaded with new version-- equation 16 was incorrect</p></details> |
| **[PRIMER: Perception-Aware Robust Learning-based Multiagent Trajectory Planner](http://arxiv.org/abs/2406.10060v3)** | 2025-04-25 | [papers-cool](https://papers.cool/arxiv/2406.10060v3) | 7 pages, 3 figures |
| **[Let's Make a Splan: Risk-Aware Trajectory Optimization in a Normalized Gaussian Splat](http://arxiv.org/abs/2409.16915v2)** | 2025-04-25 | [papers-cool](https://papers.cool/arxiv/2409.16915v2) | <details><summary>First...</summary><p>First two authors contributed equally. Project Page: https://roahmlab.github.io/splanning</p></details> |
